windows/test-app.mjs

```diff
@@ -0,0 +1,228 @@
+//@ts-check
```

Comment:
Loading module is experimental feature in Node 12, so running this script will give a warning. Any suggestion on what we should do?

Reply:
You can use `require()` instead:
```js
const path = require('path');
const fs = require('fs');
```